proto/heartbeat.go

```diff
@@ -23,6 +23,13 @@ import (
 	"time"
 )
 
+// TODO(marc): we should assert
+// var _ security.AnyUserRequest = &PingRequest{}
+// here, but we need to break cycles first.
+
+// AnyUser implements security.AnyUserRequest.
+func (*PingRequest) AnyUser() {}
```

Comment:
nah, we should use something that actually talks sql. It's also not needed. Ping specifically says it may do nothing.
We should keep this restricted to `NodeUser`. This also removes the need for `AnyUser`, it's testing only after that.